packages/injective-math/src/fp_decimal/from_str.rs

```diff
@@ -14,7 +14,7 @@ impl FromStr for FPDecimal {
     /// This never performs any kind of rounding.
     /// More than 18 fractional digits, even zeros, result in an error.
     fn from_str(input: &str) -> Result<Self, Self::Err> {
-        let sign = if input.starts_with('-') { 0 } else { 1 };
+        let sign = i8::from(!input.starts_with('-'));
```

Comment:
trying this for the second time now, the old code is more readable, I know you get this suggested from somewhere, I don't though, maybe need to fix your IDE?

Reply:
`clippy 0.1.69 (84c898d6 2023-04-16)`